client/blocks/comment-detail/comment-detail-header.jsx

```diff
@@ -82,7 +82,9 @@ export const CommentDetailHeader = ( {
 						</span>
 					</div>
 					<div className="comment-detail__author-info-element">
-						{ translate( 'on %(postTitle)s', { args: { postTitle } } ) }
+						{ translate( 'on %(postTitle)s', { args: {
+							postTitle: postTitle ? decodeEntities( postTitle ) : translate( 'Untitled' ),
```

Comment:
> Might be nice if it displayed what was in the post content, in text form, like [image] or something like that.

Oh it'd be cool if we pulled out the alt text for those sorts of posts, but I think we can tackle that edge case in another PR.